src/redis_lock/__init__.py

```diff
@@ -197,11 +231,13 @@ def __exit__(self, exc_type=None, exc_value=None, traceback=None, force=False):
         if self._lock_renewal_thread is not None:
             self._stop_lock_renewer()
         logger.debug("Releasing %r.", self._name)
-        try:
-            self._client.evalsha(UNLOCK_SCRIPT_HASH, 2, self._name, self._signal, self._id)
-        except NoScriptError:
+        is_cached, _ = _redis_eval(
```

Comment:
The logging? Not really, in ideal case it's only logged once (the first time).

Or did you ask something else?